tox.ini

```diff
@@ -0,0 +1,32 @@
+[tox]
+minversion = 4.20.0
+envlist = lint,tests,typing
+isolated_build = true
+
+[testenv]
+skip_install = true
```

Comment:
IMO, this should not be the default, but rather be set in environments where it's actually needed. Accordingly, `-e .` entries in `deps` below should be removed.

Reply:
It seems to me that some environments, `typing` and `lint`, can still have `skip_install = true`.